apps/sms/services/README.md

```diff
@@ -0,0 +1,91 @@
+# The layers in the Messages application
+
+## Terms explanation
+
+There are 3 layers whose interfaces managed by the Bridge (aka [threads.js](https://github.com/gaia-components/threads/)):
+
+* Services define high-level semantics to access the model. They are the only
```

Comment:
So it sounds like prefer there's no another js as intermediary for bridge client, and view should call the the bridge client directly. I think Oleg and I have some thought in https://github.com/mozilla-b2g/gaia/pull/30144#issuecomment-107830843 and  https://github.com/mozilla-b2g/gaia/pull/30144#issuecomment-108304433

About the naming for API wrapper/shim, I'm fine for the both name, but just wondering why not calling service in general. Because it would be run in worker eventually.

For the manager, I could image there might be some reusable lib that not necessary for bridge service, but how could we define the role of activity_handler in Oleg's activity service?